tests/core/fake_device_module.py

```diff
@@ -33,7 +33,7 @@ def _mock_with_name(name: str) -> MagicMock:
 
 
 def wrong_return_type() -> int:
-    return "0"  # type: ignore
+    return "str"  # type: ignore
```

Comment:
can we get a comment for this? where would we use this?

Reply:
This is unrelated to this change and will go away with a rebase on main. It's a result of [this change](https://github.com/DiamondLightSource/blueapi/pull/598) that made the handling of type conversion must more capable (by using the Pydantic handling rather than rolling our own).